pkgs/build-support/go/module.nix

```
@@ -153,7 +155,7 @@ let
 
     inherit (go) GOOS GOARCH;
 
-    GOFLAGS = lib.optionals (!proxyVendor) [ "-mod=vendor" ] ++ lib.optionals (!allowGoReference) [ "-trimpath" ];
+    GOFLAGS = GOFLAGS ++ lib.optionals (!proxyVendor) [ "-mod=vendor" ] ++ lib.optionals (!allowGoReference) [ "-trimpath" ];
```

```suggestion
    GOFLAGS = GOFLAGS
      ++ lib.optional (!proxyVendor) "-mod=vendor"
      ++ lib.optional (!allowGoReference) "-trimpath";
```
we can make this a little bit nicer

---

Great, thanks! :)

---

Don't agree that it is nicer. I'd prefer that this be reverted.